app/src/test/java/com/mobiledevelopmentfoodapp/FoodTest.kt

```diff
@@ -1,4 +1,4 @@
-package com.mobiledevelopmentfoodapp.com.mobiledevelopmentfoodapp
+package com.mobiledevelopmentfoodapp
```

Comment:
Changed the package structure to be more logical.

Reply:
Good.